### PR TITLE
feat(annotation): scroll into view on activeEntryId change

### PR DESCRIPTION
**Pull request by ConradJChan**
Add support for:

-  `AnnotationActivity` being scrolled into view. 
- Add support for `activeEntryId` changing triggering scroll into view logic

### CODE REVIEW

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js [hunk 1]
```diff
@@ -79,13 +83,14 @@ class ActivityFeed extends React.Component<Props, State> {
         const hasMoreItems = prevFeedItems && currFeedItems && prevFeedItems.length < currFeedItems.length;
         const didLoadFeedItems = prevFeedItems === undefined && currFeedItems !== undefined;
         const hasInputOpened = currIsInputOpen !== prevIsInputOpen;
+        const hasActiveFeedEntryIdChanged = activeFeedEntryId && prevActiveFeedEntryId;
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js:86]: Should this be `activeFedEntryId && activeFedEntryId !== prevActiveFeedEntryId`?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js [hunk 2]
```diff
@@ -79,13 +83,14 @@ class ActivityFeed extends React.Component<Props, State> {
...
 
         if ((hasLoaded || hasMoreItems || didLoadFeedItems || hasInputOpened) && activeFeedEntryId === undefined) {
             this.resetFeedScroll();
         }
 
         // do the scroll only once after first fetch of feed items
```
Comment (mickr) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js:92]: Is this comment still relevant

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js (hunk 1 above)
Comment (ConradJChan) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js:86]: I had that originally but then `scrollToActiveFeedItemOrErrorMessage` bails if `activeFeedEntryId` is falsy so I just took it out. I could go either way though, do you have a preference?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js (hunk 2 above)
Comment (ConradJChan) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js:92]: Nice catch, I'll remove the comment

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js (hunk 1 above)
Comment (jstoffan) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js:86]: My main concern is that the variable name indicates something changed, but we're not really checking for that.

**Pull request merged**